bottle.py

```diff
@@ -1527,7 +1527,7 @@ def remote_route(self):
             work if all proxies support the ```X-Forwarded-For`` header. Note
             that this information can be forged by malicious clients. """
         proxy = self.environ.get('HTTP_X_FORWARDED_FOR')
-        if proxy: return [ip.strip() for ip in proxy.split(',')]
+        if proxy: return list(map(str.strip, proxy.split(',')))
```

Comment:
IMO, the list comprehension is (a) more Pythonic, and (b) easier on the eyes. I recommend keeping it the prior way.

Reply:
I agree here.